src/cli/command_list.py

```diff
@@ -201,21 +201,18 @@ def experiments_create(
     """
     Create new experiment.
     """
+    validate_path_name("Experiment", name)
+
     cwd = Path.cwd()
-    typer.echo(f"Create experiment: '{name}' with network: '{network_name}' for application: '{application}'.")
 
-    is_app_valid, validation_message = processor.applications_validate(application)
+    # processor.applications_validate(application)
+    # TODO: Uncomment above line after application validate command is ready
+
+    processor.experiments_create(name=name, application=application.lower(),
```

Comment:
Here you lower the app name. When an experiment is created for an application with capitals it isn't working because later the application (lowercase) is not found in app config file (application_exists).
Instead of adding a lower in application_exist please think a bit where we want the translation to lower case to let all the commands for the cli work correctly, same for the network name.